source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -140,27 +146,9 @@ To install the VirtualBox kernel modules, here are the steps:
 
      clr-boot-manager update
 
+.. note:: 
 
-Install Guest Additions
------------------------
-
-The kernel modules are shipped with the ``kernel-lts`` bundle; however,
-you'll need to install the *user* Linux Guest Additions. To install the
-VirtualBox Guest Additions, follow these steps:
-
-#. Insert Guest Additions CD image using *Devices* menu
-
-   .. image:: ./figures/vbox-cd.png
-      :alt: VirtualBox CD
-
-#. Install Linux users Guest Additions::
-
-     install-vbox-lga
-
-#. Reboot your system::
-
-     reboot
-
+  :abbr:`LGA (Linux Guest Additions)` drivers are integrated in Clear Linux LTS kernel 4.14 going forward. You do not need to manually install LGA. 
```

Comment:
@miguelinux Is there still user-space content that needs to be installed by the script, or should we deprecate the script?

Reply:
@miguelinux can answer to whether the script should be deprecated?

Reply:
I have fixed the script -- the installer had changed, and we needed to take additional steps to prevent the script from rebuilding modules. Now, the script properly triggers installation of the required user-space components, and does not attempt to rebuild the kernel modules.